c7n/resources/rdscluster.py

```diff
@@ -40,7 +40,7 @@ class RDSCluster(QueryResourceManager):
     class resource_type(object):
 
         service = 'rds'
-        type = 'rds-cluster'
+        type = 'cluster'
```

Comment:
should be fine, i'm not aware of any usage of `type` in the codebase afaicr, its a histerical artifact.